prombench/manifests/prombench/benchmark/7_prometheus_remote_write.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: prometheus-sink
+  namespace: prombench-{{ .PR_NUMBER }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sink
+  template:
+    metadata:
+      namespace: prombench-{{ .PR_NUMBER }}
```

Comment:
Normally containers inherit the namespace from the Deployment.
Maybe it will work.